engine/class_modules/sc_mage.cpp

```diff
@@ -6069,6 +6084,16 @@ struct meteorite_impact_t final : public mage_spell_t
   }
 };
 
+struct arcane_rebound_t final : public arcane_mage_spell_t
```

Comment:
This needs to derive from `spell_t` since it doesn't benefit from any of the mage effects.